### PR TITLE
mimic node.js dns.resolve

### PR DESCRIPTION
**Pull request by mvayngrib**
not sure if you're still maintaining this module. 

I wanted to use it in a browserified bundle as a replacement for node's built-in dns module, and I needed **dns.resolve**

### CODE REVIEW

**Comment by indutny** [comment]
Great! I have a suggestion, though. If you are interested in working on `dns.js` - it would be great to adopt `.jscsrc` policy for Style Checking: https://github.com/indutny/bn.js/blob/master/.jscsrc . Would you mind adding it? Other than that - commits looks good to me!

**Comment by mvayngrib** [comment]
Sounds good, added and restyled

File: lib/dns.js [hunk 1]
```diff
@@ -31,7 +72,7 @@ Client.prototype.query = function query(type, host, cb) {
     ra: true
   });
 
-  if (host[host.length - 1] !== '.')
+  if (host[ host.length - 1 ] !== '.')
```
Comment (indutny) [inline lib/dns.js]: Really? Does it require this?

File: lib/dns.js [hunk 2]
```diff
@@ -68,7 +109,7 @@ Client.prototype.feed = function feed(data) {
   var seq = packet.id;
 
   // No request - ignore response
-  var req = this._reqs[seq];
+  var req = this._reqs[ seq ];
```
Comment (indutny) [inline lib/dns.js]: Same here and below.

File: lib/dns.js (hunk 1 above)
Comment (mvayngrib) [inline lib/dns.js]: yea, i think it's these:

  "requireSpacesInsideArrayBrackets": "all",
  "requireSpacesInsideObjectBrackets": "all",

i'm all for getting rid of these two rules :)

Comment (indutny) [inline lib/dns.js]: Odd, because I don't do this in `bn.js`... :(

Comment (indutny) [inline lib/dns.js]: This rule usually applies to array literals: `[ 1, 2, 3 ]`

Comment (mvayngrib) [inline lib/dns.js]: Yea, actually, I just tried cloning bn.js and ran jscs with the .jscsrc there, and it has lots of complaints. Maybe your local setup uses a different .jscsrc, like one from your home dir.

Comment (indutny) [inline lib/dns.js]: I don't have one, and tests are passing in bn.js :)

Comment (mvayngrib) [inline lib/dns.js]: Hm, seems it was an issue with jscs 1.10 https://github.com/jscs-dev/node-jscs/issues/923.

After upgrading my jscs to 1.11.3, the "space" problem has disappeared. I'll update the pull request here. However, I still see style errors for bn.js:

``` bash
~/Code/bn.js on master
$ jscs --version
1.11.3
~/Code/bn.js on master
$ jscs -c .jscsrc .
Line must be at most 80 characters at ./benchmarks/index.js :
    69 |var b5 = new sjcl(b1.toString(16));
    70 |
    71 |var a6 = new BigInteger('012345678901234567890123456789012345678901234567890', 10);
-------------------------------------------------------------------------------------------^
    72 |var b6 = new BigInteger('213509123601923760129376102397651203958123402314875', 10);
    73 |

...

Extra quotes for key at ./benchmarks/index.js :
   321 |    am1.redInvm();
   322 |  },
   323 |  'sjcl': function() {
----------^
   324 |    am5.inverseMod(prime5);
   325 |  }

Extra quotes for key at ./benchmarks/index.js :
   330 |    a1.gcd(b1);
   331 |  },
   332 |  'bigi': function() {
----------^
   333 |    a4.gcd(b4);
   334 |  },

Extra comma following the final element of an array or object literal at ./benchmarks/index.js :
   332 |  'bigi': function() {
   333 |    a4.gcd(b4);
   334 |  },
------------^
   335 |});
   336 |


48 code style errors found.
```

**Comment by mvayngrib** [comment]
I updated the PR, much fewer style changes this time :) About bn.js, see https://github.com/indutny/dns.js/pull/1#discussion_r25372190

File: lib/dns.js (hunk 1 above)
Comment (indutny) [inline lib/dns.js]: Yeah, but benchmarks are not style-checked yet. I usually do `npm test` there.

**Comment by indutny** [comment]
LGTM, thank you!

**Comment by indutny** [comment]
Landed.

**Pull request closed without merging**